orbstation/mob/simple_animal/friendly/amoung.dm

```diff
@@ -41,131 +41,98 @@
-	icon_living = "amoung_brown"
-	icon_dead = "amoung_brown_dead"
+	greyscale_colors = "#72491e"
 
 /mob/living/simple_animal/amoung/cyan
-	icon_state = "amoung_cyan"
-	icon_living = "amoung_cyan"
-	icon_dead = "amoung_cyan_dead"
+	greyscale_colors = "#39e2dd"
 
 /mob/living/simple_animal/amoung/lime
-	icon_state = "amoung_lime"
-	icon_living = "amoung_lime"
-	icon_dead = "amoung_lime_dead"
+	greyscale_colors = "#50f038"
 
 	//amoung pequeño amoung pequeño amoung pequeño amoung pequeño amoung pequeño
 	//amoung pequeño amoung pequeño amoung pequeño amoung pequeño amoung pequeño
 
-/mob/living/simple_animal/amoung/p
+/mob/living/simple_animal/amoung/pequeno
 	name = "amoung pequeño"
 	desc = "What a sooooo small creature. Probably harmless?"
-	icon_state = "ap_red"
-	icon_living = "ap_red"
-	icon_dead = "ap_red_dead"
-
-/mob/living/simple_animal/amoung/p/blue
-	icon_state = "ap_blue"
-	icon_living = "ap_blue"
-	icon_dead = "ap_blue_dead"
-
-/mob/living/simple_animal/amoung/p/green
-	icon_state = "ap_green"
-	icon_living = "ap_green"
-	icon_dead = "ap_green_dead"
-
-/mob/living/simple_animal/amoung/p/pink
-	icon_state = "ap_pink"
-	icon_living = "ap_pink"
-	icon_dead = "ap_pink_dead"
-
-/mob/living/simple_animal/amoung/p/orange
-	icon_state = "ap_orange"
-	icon_living = "ap_orange"
-	icon_dead = "ap_orange_dead"
-
-/mob/living/simple_animal/amoung/p/yellow
-	icon_state = "ap_yellow"
-	icon_living = "ap_yellow"
-	icon_dead = "ap_yellow_dead"
-
-/mob/living/simple_animal/amoung/p/black
-	icon_state = "ap_black"
-	icon_living = "ap_black"
-	icon_dead = "ap_black_dead"
-
-/mob/living/simple_animal/amoung/p/white
-	icon_state = "ap_white"
-	icon_living = "ap_white"
-	icon_dead = "ap_white_dead"
-
-/mob/living/simple_animal/amoung/p/purple
-	icon_state = "ap_purple"
-	icon_living = "ap_purple"
-	icon_dead = "ap_purple_dead"
-
-/mob/living/simple_animal/amoung/p/brown
-	icon_state = "ap_brown"
-	icon_living = "ap_brown"
-	icon_dead = "ap_brown_dead"
-
-/mob/living/simple_animal/amoung/p/cyan
-	icon_state = "ap_cyan"
-	icon_living = "ap_cyan"
-	icon_dead = "ap_cyan_dead"
-
-/mob/living/simple_animal/amoung/p/lime
-	icon_state = "ap_lime"
-	icon_living = "ap_lime"
-	icon_dead = "ap_lime_dead"
+	icon_state = "amoung_pequeno"
+	icon_living = "amoung_pequeno"
+	icon_dead = "amoung_pequeno_dead"
+
+/mob/living/simple_animal/amoung/pequeno/Initialize(mapload)
+	. = ..()
+	set_greyscale(new_config=/datum/greyscale_config/amoung_pequeno)
+
+/mob/living/simple_animal/amoung/pequeno/red
+	greyscale_colors = "#ff0033"
+
+/mob/living/simple_animal/amoung/pequeno/blue
+	greyscale_colors = "#3366ff"
+
+/mob/living/simple_animal/amoung/pequeno/green
+	greyscale_colors = "#10802d"
+
+/mob/living/simple_animal/amoung/pequeno/pink
+	greyscale_colors = "#ee55ba"
+
+/mob/living/simple_animal/amoung/pequeno/orange
+	greyscale_colors = "#f17d0e"
+
+/mob/living/simple_animal/amoung/pequeno/yellow
+	greyscale_colors = "#e9ea53"
+
+/mob/living/simple_animal/amoung/pequeno/black
```

Comment:
Do we still want all of these definitions?
I might prefer just to have it randomised on spawn if possible, or pick randomly from a preset list of colours.
In most cases I don't think someone spawning one will care about a specific colour and if they do, they can varedit

Reply:
I am all for that, would make things cleaner

Reply:
Yeah, I recommend merging them all into one mob for adults and one mob for babies. Having all of them be separately defined means that things which spawn friendly animals disproportionately pick amongi over anything else because of how many there are

Reply:
I have not considered that, that is a very good reason for merging them. I will do that asap after work.